plugins/index.ts

```diff
@@ -80,4 +94,11 @@ export const plugins = {
     toxicity: patronustoxicity,
     custom: patronuscustom,
   },
+  mistral: mistralGuardCategories.reduce(
```

Comment:
We can convert this to a single plugin like portkey.moderateContent and accept all the required categories in it